jwst/lib/set_telescope_pointing.py

```diff
@@ -194,6 +195,10 @@ def add_wcs(filename, default_pa_v3=0., siaf_path=None, engdb_url=None,
         reduce_func=reduce_func,
         **transform_kwargs
     )
+
+    if model.meta.target.type.lower() == 'moving':
```

Comment:
`lower()` will fail if `meta.target.type` is `None`, i.e not set, which might be the case for some data.

Reply:
Oh, and this is actually why the unit tests are failing.  Check for `None` first.